pysumma/preprocessing/meta/basin_extraction.R

```diff
@@ -0,0 +1,95 @@
+library(rgrass7)

```

Comment:
In my first look through it's not entirely clear where these are used. Could you point me to where these are called? I'd prefer not to introduce too many external scripts.

Reply:
This goes for all of the scripts in `preprocessing/meta/`

Reply:
Also, could you just provide some sort of header comments on what these actually do?